lib/api_repo.js

```diff
@@ -463,6 +519,38 @@ ApiRepo.prototype._buildProtos =
       }
     };
 
+/**
+ * Collect the required files from a .proto file. 'done' will be
+ * called with the list of file name within the includePath when
+ * the task has finished.
+ */
+function collectProtoDeps(protoFile, includePath, done) {
```

Comment:
nit: similarly here is a function declared after its use. In other languages, I think it's generally considered more readable to do the reverse, but if this is JS-idiomatic, that's fine.